pkg/rpctype/rpc.go

```diff
@@ -58,22 +57,11 @@ type RPCClient struct {
 }
 
 func NewRPCClient(addr string) (*RPCClient, error) {
-	var conn net.Conn
-	var err error
-	if addr == "stdin" {
-		// This is used by vm/gvisor which passes us a unix socket connection in stdin.
-		// TODO: remove this once we switch to flatrpc for target communication.
-		conn, err = net.FileConn(os.Stdin)
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		conn, err = net.DialTimeout("tcp", addr, 3*time.Minute)
-		if err != nil {
-			return nil, err
-		}
-		setupKeepAlive(conn, time.Minute)
+	conn, err := net.DialTimeout("tcp", addr, 3*time.Minute)
```

Comment:
I'm not sure I now understand how gVisor fuzzing works. Do we return `stdin:0` just to return something, but in fact we just connect from syz-executor to the syz-manager's rpc server port? Or does `stdin` have any special meaning?

Reply:
Yes.